### ISSUE TITLE
Comparison with shapeless-scalacheck?

### ISSUE THREAD

**Issue by FranklinChen** [alexarchambault/scalacheck-shapeless#2]
I'm curious how this library differences from [`shapeless-scalacheck`](https://github.com/typelevel/shapeless-contrib). Maybe you can add a note on the difference for those who already use that?

**Comment by alexarchambault** [comment]
You're right, I was thinking at the beginning to just submit scalacheck-shapeless to shapeless-contrib (https://github.com/typelevel/shapeless-contrib/issues/28), but it finally remained a separate project. I'm checking about the status of shapeless-contrib, and I think I'll add back a mention about it in the README then.

**Comment by alexarchambault** [comment]
For know, the main difference is that scalacheck-shapeless depends on more recent versions of both shapeless (2.1-RC1) and scalacheck (1.12.1). The coproduct support in scalacheck shapeless is also more finely tuned, as all cases of the coproduct are given the same weight, which doesn't seem to be the case in shapeless-contrib.
